File: Service_eHospital_DongNai_A/Implement/XepLichMo_EKipMoService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class XepLichMo_EKipMoService : BaseService<XepLichMo_EKipMo, IXepLichMo_EKipMoRepository>
    , IXepLichMo_EKipMoService
    {
    private readonly IXepLichMo_EKipMoRepository _XepLichMo_EKipMoRepository;
    public XepLichMo_EKipMoService(IXepLichMo_EKipMoRepository XepLichMo_EKipMoRepository) : base(XepLichMo_EKipMoRepository)
    {
    _XepLichMo_EKipMoRepository = XepLichMo_EKipMoRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/XepLichMo_EKipMo.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { XepLichMo_EKipMo } from './XepLichMo_EKipMo.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class XepLichMo_EKipMoService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'XepLichMo_EKipMo_Id', 'LichMo_Id', 'NhanVien_Id', 'VaiTro_Id', 'Idx'];

    List: XepLichMo_EKipMo[] | undefined;
    ListFilter: XepLichMo_EKipMo[] | undefined;
    FormData!: XepLichMo_EKipMo;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "XepLichMo_EKipMo";
    }
}

